framework/contrib/gtest/gtest/gtest.h

```diff
@@ -51,6 +51,15 @@
 #ifndef GTEST_INCLUDE_GTEST_GTEST_H_
 #define GTEST_INCLUDE_GTEST_GTEST_H_
 
+// This warning is a known issue since 2019 (https://github.com/google/googletest/issues/2224)
+// Google has yet to fix it, so we will direct clang > 9 to ignore the warnings.
+#ifdef __clang__
+#if (__clang_major__ > 9)
+// This was introduced in clang 10
+#pragma clang diagnostic ignored "-Wdeprecated-copy"
+#endif // clang > 9
+#endif
+
```

Comment:
Will this lead to ignoring bad copies in MOOSE unit test code? I'm guessing so

Reply:
I didn't consider that - this has been a longstanding issue in gtest, so I am unsure how to handle it short of diving into that code and fixing up our own copy.

Reply:
It depends on when the warnings happen. There are so many macros with gtest that *a priori*, without actually incurring the warnings during a build, it's impossible to know what macros spawn warnings. For instance, if this
```c++

TEST(ADFparserTest, JITCompile)
{
```
spawns a warning it would be obnoxious to put this diagnostic ignored pragma around every one of the macro instances.

Reply:
I don't know. What you have is probably ok. We should just remove this as soon as gtest fixes their shizz

Reply:
I could always temporarily remove and see what happens when I actually run the test suite. Do you know if we have `-Werror` turned on for that?

Reply:
I'm actually a little tempted to fix this up myself - we're already a decent amount behind HEAD and it looks like gtest doesn't accept PRs. I'll ponder over the holiday :D

Reply:
Well perhaps not.....I thought we had forked gtest, but it seems as though the last change to this file was a superficial one.